coordinator/statement_executor.go

```diff
@@ -180,11 +180,11 @@ func (e *StatementExecutor) ExecuteStatement(stmt influxql.Statement, ctx *query
 	case *influxql.ShowMeasurementsStatement:
 		return e.executeShowMeasurementsStatement(stmt, ctx)
 	case *influxql.ShowMeasurementCardinalityStatement:
-		rows, err = e.executeShowMeasurementCardinalityStatement(stmt)
+		rows, err = e.executeShowMeasurementCardinalityStatement(stmt, ctx)
```

Comment:
ctx is conventionally the first parameter of any method it is in.

Reply:
@lesam - agreed.  However, it was passed as the last parameter in some of the other functions here, so I was trying to follow their pattern.  Elsewhere in the code I pass it as normally done.

Reply:
Do you want me to switch the context to the first argument?  If so, would you like me to also change the function signature of TaskManager.ExecuteStatement(), executeExplainStatement(), executeExplainAnalyzeStatement(), executeSelectStatement(), and the caller, ExecuteStatement(), all of which have the context as the last argument?
My preference was to leave the functions called from ExecuteStatement() as is, with the context as the last argument, match that parameter order for the new signatures, and then use context as the first argument as is conventional everywhere else.

Reply:
May as well make it consistent, unless that would have a large impact on other repos.

Reply:
Will refactor if that's the consensus.